packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts

```diff
@@ -180,6 +180,20 @@ export interface VpcV2Props {
    * @default - autogenerated by CDK
    */
   readonly vpcName?: string;
+
+  /**
+   * Optional to override inferred region
+   *
+   * @default - current stack's environment region
+   */
+  readonly region?: string;
+
+  /**
+   * The ID of the AWS account that owns the VPC
+   *
+   * @default - the account id of the parent stack
+   */
+  readonly ownerAccountId?: string;
```

Comment:
this is not correct .. customer should not provide these properties. These properties in case of creating a new VPCV2 is always the stack region and stack. There is no way to add a VPC resource in some stack but created in another region or account.

Reply:
These properties are only needed in the IVPCV2 interface and all constructs that implement it.

Reply:
Update made! [5773a08](https://github.com/aws/aws-cdk/pull/31645/commits/5773a08f657c7702d8a3d028851608165f952d01)

Note: Based on in person conversation, region and ownerAccountId will always assume that of the stack when creating a vpc